.github/ISSUE_TEMPLATE/spec_issue.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Specification Review Issue
+about: Start a request for a specification release
```

Comment:
about: Start a request for a specification creation/plan/progress/release (lifecycle) review

Reply:
I am not sure about this as I don't think we really need for different stages.